packages/labs/cli/src/lib/generate/generate.ts

```diff
@@ -39,13 +42,28 @@ export const run = async (
       }
       return importer;
     });
-    await Promise.allSettled(
+    const results = await Promise.allSettled(
       importers.map(async (importer) => {
         const generator = await importer();
         // TODO(kschaaf): Add try/catches around each of these operations and
         // throw more contextural errors
-        await writeFileTree(outDir, await generator(analysis));
+        await writeFileTree(out, await generator(analysis));
       })
     );
+    // `allSettled` will swallow errors, so we need to filter them out of
```

Comment:
@justinfagnani Had suggested `allSettled` instead of `all` during an IRL code review. I agree it's marginally better to get all the errors I guess? Just realized the outer loop over multiple packages isn't parallelized though, so it's sort of halfsies.

Reply:
Yeah, I thought since we're printing errors it'd be better to print them all, unless we make a conscious decision to print just the first. `Promise.all()` just implicitly did that for us.